code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -255,7 +255,7 @@
 			u_equip(l_store, 1)
 		if (wear_id)
 			u_equip(wear_id, 1)
-		if (belt)
+		if (belt && !istype(belt, /obj/item/weapon/storage/belt))
```

Comment:
This check won't work correctly with the red ribbon arm, which sure looks like a belt to me.

Reply:
OK, I had to add some other stuff but red ribbon arm behaves properly now.